app/components/elements/forms/field_component.rb

```diff
@@ -4,18 +4,20 @@ module Elements
   module Forms
     # Base component for all form fields.
     class FieldComponent < ApplicationComponent
-      def initialize(form:, field_name:, required: false, hidden_label: false, label: nil, help_text: nil, data: {}) # rubocop:disable Metrics/ParameterLists
+      def initialize(form:, field_name:, required: false, hidden_label: false, label: nil, help_text: nil, # rubocop:disable Metrics/ParameterLists
+                     data: {}, **extra_args)
```

Comment:
This component is looking like it might be getting ripe for refactoring, judging by the growing param list. Does not need to be part of this PR, though.

Reply:
Yeah, I've been stewing over that.

Reply:
An idea I considered earlier but abandoned (because other things were more important): making the signature `initialize` look like `(form:, field_name:, **options)` and then manually writing getters for options, such as `def hidden_label = @options.fetch(:hidden_label, false)` Not sure if that's better or just different.